avalon/pipeline.py

```diff
@@ -1276,41 +1276,49 @@ def get_representation_path(representation):
 
     """
 
-    def path_from_represenation():
+    def path_from_representation():
```

Comment:
This is great, but let's also add backwards compatibility by just putting a `path_from_represenation = path_from_representation` below this function to create an alias. If you really want to go all-in, the alias could be an actual function that calls the real function, but raises and catches a deprecation warning so it can print which line it was called on in the user's code. :)

Reply:
This would be redundant right since the function declaration is inside another function? Out of that scope nobody would be calling it. Right?

Reply:
Ah yes that's right, good catch. It's safe.